global.json

```diff
@@ -1,11 +1,11 @@
 {
   "sdk": {
-    "version": "5.0.100-preview.8.20417.9",
+    "version": "5.0.100-rc.1.20452.10",
     "allowPrerelease": true,
     "rollForward": "major"
   },
   "tools": {
-    "dotnet": "5.0.100-preview.8.20417.9",
+    "dotnet": "5.0.100-rc.1.20452.10",
     "vs": {
       "version": "16.8"
     },
```

Comment:
Should the next line for `xcopy-msbuild` be updated to preview 3 instead?

Reply:
@333fred?

Reply:
(I think the answer is no because we only rebuild that xcopy package when we need to...)

Reply:
We only do it when we need to, but the likelihood is that we'll need to. @jaredpar, is there a new version? If not, can you make one?